pkg/apis/config/validation/account.go

```diff
@@ -31,6 +31,23 @@ func validateSubjects(subjects []rbac.Subject) field.ErrorList {
 	return allErrs
 }
 
+func verifySpace(account *configv1alpha1.Account) field.ErrorList {
```

Comment:
You'll have to call this function in the ValidateAccount function otherwise it is not called, is it?